scrapers/Algolia_Adultime.yml

```diff
@@ -2,33 +2,51 @@ name: "Adultime"
 sceneByURL:
   - action: script
     url:
+      - accidentalgangbang.com/en/video/
+      - adamandevepictures.com/en/video/
       - adulttime.com/en/video/
+      - AdultTimePilots.com/en/video/
       - analteenangels.com/en/video/
       - assholefever.com/en/video/
+      - BeingTrans247.com/en/video/
+      - burningangel.com/en/video/
+      - caughtfapping.com/en/video/
+      - devilsfilm.com/en/video/
+      - devilstgirls.com/en/video/
       - dpfanatics.com/en/video/
       - famedigital.com/en/video/
       - footsiebabes.com/en/video/
+      - forbiddenseductions.com/en/video/
       - girlstryanal.com/en/video/
       - hairyundies.com/en/video/
       - joymii.com/en/video/
       - kissmefuckme.com/en/video/
       - lezcuties.com/en/video/
       - massage-parlor.com/en/video/
       - milkingtable.com/en/video/
+      - modeltime.com/en/video/
       - moderndaysins.com/en/video/
       - mommysboy.com/en/video/
       - mommysgirl.com/en/video/
+      - muses.com/en/video/
       - nudefightclub.com/en/video/
       - nurumassage.com/en/video/
+      - officemsconduct.com/en/video/
       - oopsie.com/en/video/
       - outofthefamily.com/en/video/
       - peternorth.com/en/video/
+      - PinkoTGirls.com/en/video/
       - prettydirty.com/en/video/
+      - puretaboo.com/en/video/
       - soapymassage.com/en/video/
       - teensneaks.com/en/video/
+      - transfixed.com/en/video/
+      - TransgressiveFilms.com/en/video/
```

Comment:
dupe

Reply:
- the stash app matching from Scene URL text field to installed scrapers is _case-sensitive_
- the TitleCase variants are returned by Algolia API

so, if you search via Algolia API, some of the domains in the returned URLs are TitleCase, and if you only had lowercase in this list then the scraper _does not match_ and is _not usable_

Ideally, the matching of Scene URL text field to installed scraper sceneByUrl url list would be case-insensitive, and all the URLs here could just all be lowercase, but it isn't so that's why I've done what I've done here

Reply:
as above, this is a necessary dupe, as Algolia API returns this studio' scene URLs with the domain in TitleCase

Reply:
PascalCase variant removed